src/actions/metadata.js

```diff
@@ -31,6 +31,14 @@ export function updateFieldKey(namePrefix, fieldKey, newKey) {
   };
 }
 
+export function changeFieldValue(nameAttr, tempValue) {
+  return {
+    type: ActionTypes.CHANGE_FIELD_VALUE,
+    nameAttr,
+    tempValue
+  };
+}
+
```

Comment:
> I started to think that we should better move all of the things that we do on blur (trimming, auto-casting) to the put actions.

That's a nice idea. We currently do similar for outputting a JSON data file via the DataGUI.
I'll close this after you open a new PR. :+1: